index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="	https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+</head>
+<body>
+    <nav id="navbar" class="navbar navbar-expand navbar-dark">
+        <div class="container-fluid">
+            <div class="navbar-nav ms-auto">
+                <li class="nav-item"><a class="nav-link text-white mx-4 " aria-current="page" href="#">About</a></li>
+              <a class="nav-link text-white mx-4" href="#">Work</a>
+              <a class="nav-link text-white mx-4" href="#">Contact</a>
+            </div>
+        </div>
+      </nav>
+
+      <section id="welcome-section" class="text-center">
+        <h1>Hey I am Mimic</h1>
+        <p>a web developer</p>
```

Comment:
I think this is a sub header and should be put into h2 tag

Reply:
- [ ] Please explain why you are suggesting this change to the code's author. Giving reasons for changes helps the author understand the logic and concept better. It also makes it easy for the author to understand what needs to be done. 😄📗 

#### Example of a clear and easy-to-understand comment that gives reason

 > **_Kindly_** use Stylelint linters for this project. Stylelint reports bad codes in your CSS files. It helps to enforce consistent code and prevents you from making errors in your stylesheets. 👍 😎 
 
 > Kindly put the nav tag into the header tag. The nav tag is a semantic tag that should contain the logo and other navigational links to parts of the site or external sites. 
 
 #### Hint: You don't have to create another inline comment. You could edit the inline comments that you have above. 💻